### PR TITLE
Release/1.1.0 into development

### PR DESCRIPTION
**Pull request by KochTobi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,14 +12,14 @@
 
 	<groupId>life.qbic</groupId>
 	<artifactId>omero-client-lib</artifactId>
-	<version>1.1.0-SNAPSHOT</version>
+	<version>1.1.0</version>
```
Comment (wow-such-code) [inline pom.xml:15]:
```suggestion
	<version>1.2.0-SNAPSHOT</version>
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.com/qbicsoftware/omero-client-lib.svg?branch=development)](https://travis-ci.com/qbicsoftware/omero-client-lib)[![Code Coverage]( https://codecov.io/gh/qbicsoftware/omero-client-lib/branch/development/graph/badge.svg)](https://codecov.io/gh/qbicsoftware/omero-client-lib)
 
-OMERO client library, version 1.0.0-SNAPSHOT - Java-based library to connect to the OMERO server
+OMERO client library, version 1.1.0 - Java-based library to connect to the OMERO server
```
Comment (wow-such-code) [inline README.md:5]:
```suggestion
OMERO client library, version 1.2.0-SNAPSHOT - Java-based library to connect to the OMERO server
```

File: pom.xml (hunk 1 above)
Comment (KochTobi) [reply pom.xml:15]: Maybe merge it locally into development. That does not change the release/1.1.0 branch

File: README.md (hunk 2 above)
Comment (KochTobi) [reply README.md:5]: Maybe merge it locally into development. That does not change the release/1.1.0 branch

**Pull request merged**